src/PendingObjects/TestCall.php

```diff
@@ -140,7 +152,7 @@ public function skip($conditionOrMessage = true, string $message = ''): TestCall
 
         $condition = is_callable($condition)
             ? $condition
-            : function () use ($condition) { /* @phpstan-ignore-line */
+            : function () use ($condition): bool {
```

Comment:
@nunomaduro, I just wanted to check if there was a reason for not type hinting this before. 👍🏻

Reply:
With the type hint here it throws an error because it returns an integer for some reason.